test/e2e/service.go

```diff
@@ -330,7 +330,7 @@ var _ = Describe("Services", func() {
 
 		By("hitting the pod through the service's external load balancer")
 		var resp *http.Response
-		for t := time.Now(); time.Since(t) < time.Minute; time.Sleep(5 * time.Second) {
+		for t := time.Now(); time.Since(t) < podStartTimeout; time.Sleep(5 * time.Second) {
```

Comment:
But we already wait for the pod to be running (waitForPodRunning()) above?  What exactly are we waiting for here?